products/views.py

```diff
@@ -28,12 +42,31 @@ def get(self, request, product_id):
 
         return JsonResponse({'results': results}, status=200)
     
-class CategoryImageView(View):
-    def get(self, request):
-        results = [{
-                'id'   : category.id,
-                'name' : category.name,
-                'image': category.image
-            } for category in Category.objects.all()]
+class ReviewView(View):
+    @login_decorator
+    def post(self, request, product_id):
+        try:
+            signed_user = request.user
+            data        = json.loads(request.body)
+            product     = Product.objects.filter(id=product_id)
+
+            if not product.exists():
+                return JsonResponse({'message': 'PRODUCT_NOT_FOUND'}, status=404)
+
```

Comment:
```suggestion

            if not Product.objects.filter(id=product_id).exists():
                return JsonResponse({'message': 'PRODUCT_NOT_FOUND'}, status=404)
            
            product = Product.objects.get(id=product_id)
```
존재 유무 확인 후 get을 통해 특정 상품을 지정하는 것이 더 좋을 것 같습니다.

Reply:
네 감사합니다! 수정했습니다!